packages/regl-worldview/src/Worldview.js

```diff
@@ -11,49 +11,40 @@ import pickBy from "lodash/pickBy";
 import * as React from "react";
 import ContainerDimensions from "react-container-dimensions";
 
-import { CameraListener } from "./camera/index";
+import { CameraListener, DEFAULT_CAMERA_STATE } from "./camera/index";
 import type { MouseHandler, Dimensions, Vec4, CameraState } from "./types";
 import { Ray } from "./utils/Raycast";
 import { WorldviewContext } from "./WorldviewContext";
 import WorldviewReactContext from "./WorldviewReactContext";
 
 const DEFAULT_BACKGROUND_COLOR = [0, 0, 0, 1];
 
-type SharedProps = {|
+export type BaseProps = {|
   backgroundColor?: Vec4,
   hitmapOnMouseMove?: boolean,
-  hideDebug?: boolean,
+  showDebug?: boolean,
   children?: React.Node,
 
+  cameraState: CameraState,
+  defaultCameraState: CameraState,
+  // if onCameraStateChange is present, Worldview is controlled, otherwise it's uncontrolled
```

Comment:
We should probably use the presence of `cameraState` to determine whether it should be a controller component, to stay consistent with built-in React components. For example, when rendering an `<input value="test"/>` then it is a controlled component, even if we don't specify an `onChange` handler.